docs/en/metadata.yaml

```diff
@@ -1,48 +1,72 @@
 navigation:
 
-- title: Settings
+- title: Basics
   children:
 
-  - location: settings/animation-settings.md
```

Comment:
There have been some spaces deleted in the file which changes the indenting and breaks the file.
The file can be validated at https://codebeautify.org/yaml-validator if you want to check yourself.

The child items need two spaces before the `-`, and `title` should line up with `location`

Reply:
@WillMoggridge thanks for reviewing. I will take a look and update based on your feedback, I didn't realise there was a particular layout to this file.